jenkins-pipelines/features-dns-cluster-5min.jenkinsfile

```diff
@@ -9,5 +9,5 @@ longevityPipeline(
     test_name: 'build_cluster_test.BuildClusterTest.test_use_public_dns_names',
     test_config: 'test-cases/features/dns-cluster-5min.yaml',
 
-    timeout: [time: 60, unit: 'MINUTES']
+timeout: [time: 60, unit: 'MINUTES']
```

Comment:
Mistaken change.

Reply:
Corrected

Reply:
The only change that will be useful here is deletion of the line 12 with the "timeout" parameter.
Why? Because it is not used.
Proof is in the `vars/longevityPipeline.groovy` file which gets called here.